README.md

```diff
@@ -43,11 +43,13 @@ import {
 const OverviewContent = ({ entity }: { entity: Entity }) => (
   <Grid container spacing={3} alignItems="stretch">
     ...
-    {isDatadogWidgetAvailable(entity) && (
-      <Grid item>
-        <DatadogGraphWidget entity={entity} />
-      </Grid>
-    )}
+    <EntitySwitch>
+      <EntitySwitch.Case if={e => Boolean(isDatadogGraphAvailable(e))}>

```

Comment:
It looks like `isDatadogGraphAvailable` already calls `Boolean` before it returns so this can be simplified.

```suggestion
      <EntitySwitch.Case if={isDatadogGraphAvailable}>
```

https://github.com/RoadieHQ/backstage-plugin-datadog/blob/4e66aeaf30158b8129f771df8d9d3a85f1d683e7/src/plugin.ts#L18